integrations/tendermint-bsync/src/utils/merkle.ts

```diff
@@ -7,10 +7,7 @@ import { dataItemToSha256, generateMerkleRoot } from '@kyvejs/sdk';
 // Trustless API to serve block and block results independently.
 export function createHashesFromTendermintBundle(bundle: any[]): Uint8Array[] {
```

Comment:
Could you change to:
`export function createHashesFromBundle(bundle: any[]): Uint8Array[] {
  return bundle.map(dataItem => dataItemToSha256(dataItem))
}`